vizro-core/src/vizro/_vizro.py

```diff
@@ -44,10 +44,13 @@ def build(self, dashboard: Dashboard):
         Returns:
             Vizro: App object
         """
+        data_manager._cache.init_app(self.dash.server, config=data_manager._cache.config)
```

Comment:
```suggestion
        data_manager._cache.init_app(self.dash.server)
```
I guess that doesn't work?

Reply:
Remove `config=data_manager._cache.config` from here, as it's duplicate